packages/math/src/pool/concentrated/__tests__/quotes.spec.ts

```diff
@@ -719,16 +719,18 @@ describe("calcOutGivenIn matches chain code", () => {
       ];
       const curSqrtPrice = new BigDec("70.710678118654752441");
       const swapFee = new Dec("0");
-      expect(
+      try {
```

Comment:
FYI there's `expect().toThrow()` syntax for this

Reply:
Done